antissh.py

```diff
@@ -24,11 +24,17 @@
 NICKNAME = config.get('host', 'nickname', fallback='antissh')
 MODES = config.get('host', 'modes', fallback='')
 KLINE_CMD_TEMPLATE = config.get('host', 'kline_cmd', fallback='KLINE 86400 *@{ip} :Vulnerable SSH daemon found on this host.  Please fix your SSH daemon and try again later.\r\n')
+OUTGOING_IPPORT = (config.get('target', 'outgoing_ip', fallback='::'), 0)
```

Comment:
I would name this option something as simple as `bindhost`, and remove the port reference since it isn't really relevant